example/docker-compose.yml

```diff
@@ -1,17 +1,5 @@
 name: wikibase-suite-example
 
-x-common-variables: &wikibase_variables
```

Comment:
This extra common Yaml variable no longer need to be created as these parameters are passed only to the `wikibase` service which then generates a `LocalSettings.php` on first run. `wikibase-jobrunner` also uses that volume so gets all its settings then from the `LocalSettings.php` file.